src/Microsoft.Health.ControlPlane.Core/Features/Rbac/IdentityProvider.cs

```diff
@@ -41,8 +45,8 @@ public IdentityProvider(string name, string authority, IReadOnlyList<string> aud
         [JsonProperty("audience")]
         public virtual IReadOnlyList<string> Audience { get; protected set; }
 
-        [JsonProperty("version")]
-        public virtual string Version { get; protected set; }
+        [JsonProperty("etag")]
+        public virtual string VersionTag { get; protected set; }
```

Comment:
Why is there a name mismatch?

Reply:
This is because there is already a eTag on the derived class that is deserialied from the document.